src/eCRC_qaautomation/automationtest/smoketests/success_paymentcancel_spec.js

```diff
@@ -1,191 +1,256 @@
 //import { browser, element, by } from "protractor"
 
-require('dotenv').config();
+require("dotenv").config();
 
-var bcscRedirectPage = require('../../pageobjectfactory/bcscredirectpage');
+var bcscRedirectPage = require("../../pageobjectfactory/bcscredirectpage");
 
-var landingPage = require('../../pageobjectfactory/landingpage');
+var landingPage = require("../../pageobjectfactory/landingpage");
 
-var bcscRedirectPage = require('../../pageobjectfactory/bcscredirectpage');
+var bcscRedirectPage = require("../../pageobjectfactory/bcscredirectpage");
 
-var orgVerificationPage = require('../../pageobjectfactory/orgverificationpage');
+var orgVerificationPage = require("../../pageobjectfactory/orgverificationpage");
 
-var termsOfUsePage = require('../../pageobjectfactory/termsofusepage');
+var termsOfUsePage = require("../../pageobjectfactory/termsofusepage");
 
-var bcServicesCardLandingPage = require('../../pageobjectfactory/bcservicescardlandingpage');
+var bcServicesCardLandingPage = require("../../pageobjectfactory/bcservicescardlandingpage");
 
-var bcServicesCardLoginPage = require('../../pageobjectfactory/bcservicescardloginpage');
+var bcServicesCardLoginPage = require("../../pageobjectfactory/bcservicescardloginpage");
 
-var bcscConsentPage = require('../../pageobjectfactory/bcscconsentpage');
+var bcscConsentPage = require("../../pageobjectfactory/bcscconsentpage");
 
-var consentPage = require('../../pageobjectfactory/consentpage.js');
+var consentPage = require("../../pageobjectfactory/consentpage.js");
 
-var applicationFormPage = require('../../pageobjectfactory/applicationformpage');
+var applicationFormPage = require("../../pageobjectfactory/applicationformpage");
 
-var paymentPage = require('../../pageobjectfactory/paymentpage');
+var paymentPage = require("../../pageobjectfactory/paymentpage");
 
-var informationReviewPage = require('../../pageobjectfactory/informationreviewpage');
+var informationReviewPage = require("../../pageobjectfactory/informationreviewpage");
 
-var testInput = require('../../input/success');
+var testInput = require("../../input/success");
 
-describe('success', function(){
+describe("success", function() {
+  beforeEach(function() {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
+  });
```

Comment:
are we not allowed to use arrow functions here?
```suggestion
describe("success", () => {
  beforeEach(() => {
    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
    jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
  });
```